### DISCUSSION TITLE
오류코드 1000

### DISCUSSION THREAD

**Discussion by smilejsu82** [naverz/zepeto-studio-kor#587] [Issue Report (Legacy)]
![image](https://user-images.githubusercontent.com/50188795/161765295-39a01316-139f-48ac-8000-526de4a2d3b7.jpeg)

월드 : 1.1.5
제페토 버전 : 3.11.0
테스트 기기 : 갤럭시 s10 5g
서버 : Development 
네트워크 환경 : wifi

아래는 동일 환경 정상 동작 합니다 

![image](https://user-images.githubusercontent.com/50188795/161766036-044dcb48-5df9-4ae0-8a02-564f17ffb220.png)

월드 : 1.1.5
제페토 버전 : 3.11.0
테스트 기기 : 아이폰 13 pro
서버 : Development 
네트워크 환경 : wifi

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 오류코드 1000은 메세지 내용처럼 네트워크 오류입니다.
현재도 같은 현상이 나타나는지 확인 부탁 드리며
작업중인 다른 월드가 있으시다면 다른 월드에서도 확인 부탁 드립니다.
